CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ add_library(${CMAKE_PROJECT_NAME} MODULE)
 # author of the plugin (in the plugin's metadata itself and in the installers)
 set(PLUGIN_AUTHOR "Your Name Here")
 
+# Replace 'http://www.example.com` with a link to the website of your plugin or repository
+set(PLUGIN_WEBSITE "http://www.example.com")
```

Comment:
Unless there is a strong reason not to, use `https` in the comment and the actual variable value.